students/Gabe_Lamberth/lesson02/grid_printer.py

```diff
@@ -0,0 +1,61 @@
+def plus(): 
+    return('+ ')
+
+def dash():
+    return('- ') 
+
+def bar():
+    return '| '
+
+def printGrid(size):        
+    # Print arbitrary grid size like image in assignment part 2
+    #using variable space to make arbitrary gridsize with value size
+    #using round to round up
+    space = round(size/2)
+    
+    #I messed around with the formatting so much, that once I found
+    # something that worked, I left good enough alone
+    print('{}{}'.format((plus()+ dash()*space),(plus()+dash()*space),), end='+')
+    print()
+    for l in range(space): print('{}'.format(bar()+'  '*space)*3)    
```

Comment:
Don't put the execution of the for loop on the same line.  Makes it hard to read